.github/workflows/ci.yml

```
@@ -0,0 +1,237 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - '*.x'
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - main
+  schedule:
+    # Weekly Monday 9AM build
+    # * is a special character in YAML so you have to quote this string
+    - cron: '0 9 * * 1'
+
+env:
+  CRDS_SERVER_URL: "https://roman-crds-test.stsci.edu"
+  CRDS_PATH: "/tmp/crds_cache"
+  CRDS_CLIENT_RETRY_COUNT: 3
+  CRDS_CLIENT_RETRY_DELAY_SECONDS: 20
+
+jobs:
+  style:
+    name: Code style checks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.0.2
+      - uses: actions/setup-python@v4.1.0
+        id: python
+        with:
+          python-version: 3.9
+      - uses: actions/cache@v3.0.5
+        with:
+          path: ${{ env.pythonLocation }}
+          key: style-${{ runner.os }}-${{ env.pythonLocation }}-${{ hashFiles('**/pyproject.toml', '**/setup.*') }}
+      - run: pip install pyproject-flake8
+      - run: pip freeze
+      - run: pflake8
```

this should be `flake8`, I have another PR to address this issue in `romancal`
```suggestion
      - run: pip install flake8
      - run: pip freeze
      - run: flake8
```

---

Fixed.